setup.py

```diff
@@ -73,15 +73,19 @@ def cpp_flag(compiler):
 
 class BuildExt(build_ext):
     """A custom build extension for adding compiler-specific options."""
+    native_flag = '-march=native'
```

Comment:
Can you please name it `compiler_flag_native`?

Reply:
Done